CONTRIBUTING.md

```diff
@@ -24,33 +24,73 @@ The ILP RFC process attempts to be easy to use but also rigorous enough that the
+
+#### Publishing a Proposal
+To publish a new Proposal, submit a Pull Request to the [RFC Proposals repo](https://github.com/interledger/rfcs/proposals) with a new Markdown file. The file MUST follow the naming convention `0000-title`, where title is a lower case title with spaces replaced by hyphens (`-`), and have front-matter (similar to GitHub pages rendered from MarkDown) specifying at least a `title`, `type`, and `draft` (an integer, starting at 1 and incrementing with each revision of the RFC). The `type` MUST have the value `proposal`.
 
 Example:
-```
----
+```markdown
+\---
 title: The Interledger Architecture
+type: proposal
 draft: 1
----
+\---
 # Interledger Architecture
 
 Lorum Ipsum etc
 ```
 
-2. One of the project maintainers will review the submission and assign an RFC number to the document, making a follow up commit to the PR which renames the file and folder as appropriate.
+### Working Drafts 
+A _Working Draft_ is an RFC that has gained significant consensus support from the Interledger community. Working Drafts differ from Proposals in that edits to the document undergo significant community review and are not merged until there is broad consenus among the editors of the RFC, and the broader Interledger community.
+
+#### Publishing a Working Draft
+Assuming there is consensus to publish, one of the project maintainers will review the submission and assign an RFC number to the document, making a follow up commit to the PR which renames the file as appropriate. 
+
+The maintainer will move the document into the [RFC Working Drafts repo](https://github.com/interledger/rfcs/working-drafts) and update the `type` to be `working-draft`. 
+
+Subsequent updates to the document will trigger a publishing workflow that creates an HTML rendered version of the document that contains a permalink to the draft. Each revision must increment the `draft` number in the front-matter or the publishing will fail.
 
-3. Subsequent updates to the document will trigger a publishing workflow that creates an HTML rendered version of the document that contains a permalink to the draft. Each revision must increment the `draft` number in the front-matter or the publishing will fail.
+Example:
+```markdown
+\---
+title: The Interledger Architecture
+type: working-draft
+draft: 2
+\---
+# Interledger Architecture
 
-4. These Interledger RFCs are living documents that start as Working Drafts and are iterated on until they are considered stable.
+Lorum Ipsum etc
+```
 
-5. *Working Drafts* have an RFC number AND a draft number starting at 1 and increasing by 1 each time the content is changed.
+### Candidate Specifications
+A _Candidate Specification_ is a document that was previously a Working Draft RFC that was considered stable enough by the community such that no further changes are required. Once an RFC becomes a Candidate Specification, no further substantive changes are allowed under the same RFC number, and the draft number changes to FINAL.
```

Comment:
A candidate spec should not have a draft number of FINAL. Revisions are allowed if they are not substantive.

Reply:
Not opposed, but the current document says:

> assuming there is consensus to publish, the document becomes a *Candidate Specification* and no further substantive changes are allowed under the same RFC number. (The draft number changes to `FINAL`).

Is the current doc incorrect?

Reply:
Bump @adrianhopebailie

Reply:
> Is the current doc incorrect?

No, but I was suggesting this would be better. WDYT?

Reply:
I agree - RFCs that have graduated to become a Candidate Spec shouldn't really change in any substantive manner. If something needs to change, it should probably be versioned (e.g., `BTP2` vs `BTP3`). 

That said, we should allow for errata so I adjusted this to include a way to indicate an edit -- it's basically just an errata version number in the "front-matter" of the RFC. That way, anyone looking can see that something was changed since they last looked.

Happy to entertain an alternative system though - in the IETF they actually have a separate document containing approved errata, but this felt like overkill to me.

Reply:
Resolving this for now - plz post a new comment if you want to suggest a change.